.gitignore

```diff
@@ -5,7 +5,7 @@ Car_owners/car_owners/__pycache__/
 
 # Django stuff:
 *.log
-local_settings.py
+Car_owners/data
```

Comment:
why, what is that?

Reply:
Directory "data" is created, when POSTGRESQL container starts. It contains various data connected with database. It is not needed to start application and uses a lot of memory, thus it should to be ignored in repo.